cypress/e2e/tests/pages/user_menu/account-api-keys.spec.ts

```diff
@@ -0,0 +1,92 @@
+import HomePagePo from '~/cypress/e2e/po/pages/home.po';
+import UserMenuPo from '@/cypress/e2e/po/side-bars/user-menu.po';
+import AccountPagePo from '~/cypress/e2e/po/pages/account-api-keys.po';
+import CreateKeyPagePo from '~/cypress/e2e/po/pages/account-api-keys-create_key.po';
+import PromptRemove from '@/cypress/e2e/po/prompts/promptRemove.po';
+
+const userMenu = new UserMenuPo();
+const accountPage = new AccountPagePo();
+const createKeyPage = new CreateKeyPagePo();
+const apiKeysList = accountPage.list();
+
+describe('User can make changes to their account', () => {
+  beforeEach(() => {
+    cy.login();
+  });
+
+  it('Can navigate to Account and API Keys page', () => {
```

Comment:
Bit split on this one. It's good to have granular tests, however in the interests of avoiding super long test runs can it be folded in to the next test?

Reply:
yes, that's a good call. updated